index.es.md

```diff
@@ -0,0 +1,244 @@
+---
+# Translation instructions are after the "#" character in this first section. (They are comments that do not show up in the web page.)
+title: Introducción a la Accesibilidad Web   # Do not translate "title:". Do translate the text after "title:".
+lang: es   # Change "en" to the translated language shortcode from https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry
+last_updated: 2019-06-26   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+translators:
+- name: "Jorge Rumoroso"   # Replace @@ with translator name
+  link: "https://twitter.com/rumoroso"
+# contributors: #Add one -name: line for every contributor
+# - name: "Contributor Name Here"
+permalink: /fundamentals/accessibility-intro/es   # Add the language shortcode to the end; for example /fundamentals/accessibility-intro/fr
+ref: /fundamentals/accessibility-intro/   # Do not change this
+layout: default
+github:
+  repository: w3c/wai-intro-accessibility
+  branch: gh-pages
+  path: index.es.md   # Add the language shortcode to the middle of the filename, for example index.fr.md
+footer: >   # Translate all the words below, including "Date:" and "Editor:". Do not change these dates.
+  <p><strong>Fecha:</strong> Actualizado a 5 June 2019. Primera publicación en Febrero 2005.</p>
+  <p><strong>Editor:</strong> <a href="http://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Desarrollado por Grupo de Trabajo de Educación y Divulgación (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+# Read Translations Notes at https://github.com/w3c/wai-intro-accessibility/blob/gh-pages/README.md
+# end of translation instructions
+---
+
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Summary" class="full" %}
+{:/}
+
+Cuando los sitios y herramientas web están bien diseñados y codificados, las personas con discapacidad pueden utilizarlos. Sin embargo, en la actualidad muchos sitios y herramientas están desarrollados incluyendo barreras de accesibilidad que dificultan o imposibilitan su uso por parte de algunas personas.
+
+Hacer la web accesible beneficia tanto a las personas, como a las empresas y a la sociedad en general. Los estándares web internacionales definen lo que la accesibilidad necesita.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenido de la Página" class="full" %}
+{:/}
+
+<ul>
+<li><a href="#context">Accesibilidad en Contexto</a></li>
+<li><a href="#what">Qué es la Accesibilidad Web</a></li>
+<li><a href="#important">La Accesibilidad es Importante para las Personas, Empresas y Sociedad</a></li>
+<li><a href="#making">Hacer la Web Accesible</a></li>
+<li><a href="#evaluate">Evaluar Accesibilidad</a></li>
```

Comment:
I would use "Evaluar la Accesibilidad", or even better "Evaluando la Accesibilidad". Anyway, the article ("la") is missing there.